conbench/app/_endpoint.py

```diff
@@ -125,6 +138,9 @@ def redirect(self, endpoint, **kwargs):
         return f.redirect(f.url_for(endpoint, **kwargs))
 
     def render_template(self, template, **kwargs):
+        # inject/overwrite
+        kwargs["version_string_footer"] = VERSION_STRING_FOOTER
```

Comment:
This was the simplest / most immediate way I thought of to affect all views (they all have in common that they use this `render_template()` method. There might be more elegant ways.